### DISCUSSION TITLE
Proyecto.

### DISCUSSION THREAD

**Discussion by JulianaCadavid** [rramosp/ai4eng.v1#71] [00 - General]
Profesor, tengo varias opciones para el proyecto, me podría recomendar la mejor, son las siguientes:
* https://www.kaggle.com/c/movie-review-sentiment-analysis-kernels-only/code
* https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification/code
* https://www.kaggle.com/c/womens-machine-learning-competition-2019/data
* https://www.kaggle.com/c/osic-pulmonary-fibrosis-progression/overview

**Comment by rramosp** [comment]
hola juliana,

el tercero (NCAA Womens competition) es el que me parece más abordable.

los dos primeros son de procesamiento de texto, que también son
interesantes pero tendrías que usar alguna técnica de procesamiento de
texto.

el último es también interesante pero tendrías que mirar alguna técnica de
procesamiento de imágenes.

ambas cosas quedan un poco fuera del ámbito de este curso, pero una primera
aproximación para lo necesario en el proyecto tampoco requiere cosas tan
complejas

si quieres lo podemos hablar en cualquiera de las sesiones sincrónicas

saludos
raúl



El jue, 25 nov 2021 a las 9:34, JulianaCadavid ***@***.***>)
escribió:

> Profesor, tengo varias opciones para el proyecto, me podría recomendar la
> mejor, son las siguientes:
>
>    -
>    https://www.kaggle.com/c/movie-review-sentiment-analysis-kernels-only/code
>    -
>    https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification/code
>    -
>    https://www.kaggle.com/c/womens-machine-learning-competition-2019/data
>    - https://www.kaggle.com/c/osic-pulmonary-fibrosis-progression/overview
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/71>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZII23BF7OXMY4J7MWFTUNZCQDANCNFSM5IYWACLQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by JulianaCadavid** [comment]
Buenos días profe, lo que pasa es que no quedé cómoda con ninguna base de datos ya que el monitor me dio otras recomendaciones, me podría recomendar algunas de estas: 
https://www.kaggle.com/c/mercari-price-suggestion-challenge/overview/evaluation
https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge/code
https://www.kaggle.com/c/talkingdata-adtracking-fraud-detection/data?select=train_sample.csv
https://www.kaggle.com/c/santander-value-prediction-challenge/data
https://www.kaggle.com/c/santander-customer-transaction-prediction/data
Espero que alguna de estas si sea la ideal para el curso. Saludos.
